react-components/src/hooks/useSearchAssetsMapped360Annotations.tsx

```diff
@@ -87,18 +87,18 @@ async function get360AnnotationAssets(
   image360Annotations: AnnotationModel[],
   sdk: CogniteClient
 ): Promise<Image360AnnotationMappedAssetData[]> {
-  const annotationMapping = image360Annotations.map((annotation) => {
-    const assetId = getAssetIdOrExternalIdFromImage360Annotation(annotation);
-    if (assetId === undefined) {
-      return undefined;
-    }
-    return {
-      assetId,
-      annotationId: annotation.id
-    };
-  });
-
-  const filteredAnnotationMappings = filterUndefined(annotationMapping);
+  const filteredAnnotationMappings = image360Annotations
+    .map((annotation) => {
+      const assetId = getAssetIdOrExternalIdFromImage360Annotation(annotation);
+      if (assetId === undefined) {
+        return undefined;
+      }
+      return {
+        assetId,
+        annotationId: annotation.id
+      };
+    })
+    .filter(isDefined);
 
```

Comment:
The PR seems like more related to exposing things for rule based coloring but also has some refactoring related to the 360s... Is it intentional?
Maybe it is better to rephrase the PR title to also mention these other changes perhaps?

Reply:
Good point - changed the title. (Yes, it is intentional. It is a little messy, but I want the bar for doing improvements in the code to be low, so that we don't require separate PR's for every change)